Sky Atlas/Sky Atlas.html

```diff
@@ -8953,6 +8953,19 @@ <h1>Sections & Primary Docs</h1>
             Slope 2: 300%• Reserve State: Active• Collateral: Yes• Borrowing: Yes• Isolated Collateral: No•
             Isolated Borrowing: No• Siloed Borrowing: No• Flash Loan Enabled: Yes</td>
         </tr>
+        <tr>
+          <td>
+            <dfn>A.3.9
+              - Measures For Endgame Transition - SparkLend Risk Parameters - USDS Risk Parameters</dfn>
+          </td>
+          <td>Measures For Endgame Transition - SparkLend Risk Parameters - USDS Risk Parameters</td>
+          <td>Core</td>
+          <td>The current USDS risk parameters are:• LTV: 0%• Liquidation Threshold: 0%• E-mode Category: N/A•
+            Liquidation Bonus: 10%• Reserve Factor: 0%• Supply Cap: Unlimited• Borrow Cap: Unlimited•
+            Optimal Utilization: 95%• Isolated Debt Ceiling: N/A• Base Rate: SSR + 0.25%• Slope 1: 0%•
```

Comment:
```suggestion
            Optimal Utilization: 95%• Isolated Debt Ceiling: N/A• Base Rate: SSR + 0.5%• Slope 1: 0%•
```

Increased in https://vote.makerdao.com/executive/template-executive-vote-eth-and-wsteth-oracle-migration-rate-changes-smart-burn-engine-parameter-update-bug-bounty-payout-aligned-delegate-compensation-atlas-core-development-payments-integration-boost-top-up-spark-proxy-spell-march-20-2025